src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+
+//require('dotenv').config()
+const API_KEY = process.env.REACT_APP_API_KEY
+
+export default function Search({ onSubmit }) {
+  const [options, setOptions] = useState([])
+  const [query, setQuery] = useState('')
+  const [selectedOption, setSelectedOption] = useState(null)
+  useEffect(() => {
+    setSelectedOption(null)
+    fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
+      .then(res => res.json())
+      .then(data => setOptions(data.results))
+  }, [query])
+  return (
+    <section>
+      <Form onSubmit={handleSubmit}>
+        <label>
+          Selected Game: {selectedOption?.name || 'None'}
```

Comment:
I wouldn't put this in the label but maybe as a heading or something like that? The content of the label should describe what the `input` is for, for example: `Search for game` or something like that.